src/index.js

```diff
@@ -31,7 +35,22 @@ function startElectron (config, ports) {
         args = debugPortsArgs.concat(extraArgs);
     }
 
-    spawn(cmd, args, { stdio: 'ignore' });
+    var proc = spawn(cmd, args, { stdio: ['ignore', 'pipe', 'pipe'] });
+
+    proc.stdout.on('data', (buf) => {
+        var str = String(buf);
+        if (str[str.length - 1] === '\n') {
+            str = str.slice(0, str.length - 1);
+        }
+        STDOUT_LOGGER(str);
+    });
+    proc.stderr.on('data', (buf) => {
+        var str = String(buf);
+        if (str[str.length - 1] === '\n') {
```

Comment:
We can use [lodash.trimEnd(str, '\n')`](https://lodash.com/docs/4.17.15#trimEnd) here.